src/test/java/org/mule/extension/sftp/PostActionGroupTestCase.java

```diff
@@ -6,11 +6,33 @@
  */
 package org.mule.extension.sftp;
 
+import static org.junit.Assert.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 import static org.mule.extension.sftp.AllureConstants.SftpFeature.SFTP_EXTENSION;
```

Comment:
(Thinking loud) We need to automate the validation of the order of imports to forget about this issue... it is not important and there is no point in wasting time correcting these things.

Reply:
Yes, sorry, I will pay attention to the import order. 
I directly use the autocomplete when import. 
I will update it. 

I run testing every time before I commit the code.
If the import order can be adjusted automatically when compiling the code, it will be convenient.